docs/pages/deployment/production-configuration.rst

```diff
@@ -31,61 +27,38 @@ You can determine the intended public by observing the first part of the URL:
 * ``/internal`` is meant for XIS application integration and administrators.
 
 It's advisable to make sure internal endpoints aren't reachable from public networks.
-You can achieve this by binding it to a different port, by using a reverse proxy, or both.
-Use the ``address`` property of a HTTP binding to expose it on a different port:
+Make sure you don't accidentally expose it through your reverse proxy.
 
 .. code-block:: yaml
 
     http:
-      # The following is the default binding which endpoints are bound to,
-      # which don't have an alternative bind specified under ``alt``. Since it's a default it can be left out or
-      # be used to override the default bind address.
-      # If you specify an alt bind without address, it binds to the default HTTP interface, but
-      # the configuration of the default bind will NOT apply to the alt bind. If required, you need to repeat the config (e.g. cors/auth) for the alt bind.
-      default:
-        address: :1323
-      alt:
-        # The following binds all endpoints starting with ``/internal`` to ``internal.lan:1111``
-        internal:
-          address: internal.lan:1111
-          auth:
-            type: token_v2
-            authorizedkeyspath: /opt/nuts/authorized_keys
-            audience: nuts-node.example.com
-        # The following binds all endpoints starting with ``/public`` to ``:443`` and enables TLS for it
-        public:
-          address: :443
-          # Given we bind on port 443, we probably want to enable TLS for this endpoint
-          tls: server
-          # The following enables cross-domain requests (CORS) from irma.vendor.nl
-          cors:
-            origin:
-              - irma.vendor.nl
-
-Cross Origin Resource Sharing (CORS)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-In some deployments CORS can be required for the public IRMA authentication endpoints when the user-facing
-authentication page is hosted on a (sub)domain that differs from Nuts Node's IRMA backend. CORS can be enabled on a
-specific HTTP interface by specifying the domains allowed to make CORS requests as ``cors.origin`` (see the example above).
-Although you can enable CORS on the default endpoint it's not advised to do so in a production environment,
-because CORS itself opens up new attack vectors on node administrators.
+      # The following is the default configuration of the HTTP interfaces.
+      internal:
+        address: 127.0.0.1:8081
+        # This enables API authentication on /internal endpoints
+        auth:
+          type: token_v2
+          authorizedkeyspath: /opt/nuts/authorized_keys
+          audience: nuts-node.example.com
+      external:
```

Comment:
maybe public is better to signal internet accessibility?

Reply:
Yes, I like it better. I still don't like "internal" much, ideas?